lib/d3_version.h.in

```diff
@@ -20,8 +20,8 @@
 
 /* This file is autogenerated from lib/d3_version.h.in */
 
-#define D3_MAJORVER @PROJECT_VERSION_MAJOR@
-#define D3_MINORVER @PROJECT_VERSION_MINOR@
-#define D3_BUILD @PROJECT_VERSION_PATCH@
```

Comment:
As HogMaker/HogEdit is integral part of D3 project, these changes is not required. Please exclude relevant changes from PR.